fast64_internal/render_settings.py

```diff
@@ -14,8 +15,73 @@ def on_update_sm64_render_settings(self, context: bpy.types.Context):
 
 
 def on_update_oot_render_settings(self, context: bpy.types.Context):
-    # TODO: Update render properties from selected OOTLightProperty
-    pass
+    renderSettings: "Fast64RenderSettings_Properties" = context.scene.fast64.renderSettings
+    if renderSettings.ootSceneObject is None or not renderSettings.useObjectRenderPreview:
+        return
+    header = ootGetSceneOrRoomHeader(
+        renderSettings.ootSceneObject,
+        renderSettings.ootSceneHeader,
+        False,
+    )
+    if header is None:
+        return
+    lMode = header.skyboxLighting
+    if lMode == "0x01" or (lMode == "Custom" and not renderSettings.ootForceTimeOfDay):

```

Comment:
are there any dicts or variables that could be used instead of `"0x01"` for `lMode`? looking at this i'm not sure what the mode represents

Reply:
0 is time-of-day lighting and 1 is lighting where one set of light settings is used at a time (used for indoors). As far as I can tell from the current variable names etc. in the decomp, these are the only options for this variable (there's no point in letting it be set to something other than 0 and 1 with "custom"), though there certainly are sometimes reasons to include more than 4 and 1 lighting settings respectively. Are you saying that I should change the enum values here from "0x00" and "0x01" to more descriptive words? I didn't set those enum values in the first place, do we want to break this in people's scenes?